### PR TITLE
feat(manage-profiles-screen): UX & navigation

### PR DESCRIPTION
**Pull request by VladislavRiabko**
- add ManageProfiles screen 
- add UserImageTitle component

### CODE REVIEW

File: app/screens/Menu/ManageProfiles/ManageProfiles.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  const childNickname = useAppSelector(
+    state => state.user.child?.nickname,
+  ) as string;
+
+  const childAvatar = useAppSelector(
+    state => state.user.child?.avatar as keyof typeof IMAGES,
+  );
+  const parentAvatar = useAppSelector(
+    state => state.user.parent?.avatar as keyof typeof IMAGES,
+  );
```
Comment (Reykjavik151) [inline app/screens/Menu/ManageProfiles/ManageProfiles.tsx]: Refactor it with common interface
+ make separate images for avatars @emberglaz

File: app/screens/Menu/ManageProfiles/ManageProfiles.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            <UserImageTitle
+              title={parentNickname}
+              image={parentAvatar}
+              onPress={_.noop}
+            />
+            <UserImageTitle
+              title={childNickname}
+              image={childAvatar}
+              onPress={_.noop}
+            />
```
Comment (Reykjavik151) [inline app/screens/Menu/ManageProfiles/ManageProfiles.tsx]: TODO's for all `_.noop` expressions

**Review by Reykjavik151 — CHANGES_REQUESTED** [review]

File: app/assets/images/index.ts [hunk 3]
```diff
@@ -10,3 +10,12 @@ export const IMAGES = {
...
 };
+
+// TODO: change to correct images
+export const AVATAR_IMAGES = {
+  LOGO: require('./beyondCodelineLogo.png'),
+  WHITE_PENCIL: require('./whitePencil.png'),
+  WHITE_BACK_ARROW: require('./whiteBackArrow.png'),
+};
+
+export type AvatarsNames = keyof typeof AVATAR_IMAGES;
```
Comment (emberglaz) [inline app/assets/images/index.ts]: move to utils
create enum
`enum AvatarsNameType {Logo = 'LOGO'}`

File: app/screens/Menu/ManageProfiles/ManageProfiles.tsx [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+  ) as string;
+  const childNickname = useAppSelector(
+    state => state.user.child?.nickname,
+  ) as string;
+
+  const childAvatar = useAppSelector(
+    state => state.user.child?.avatar as AvatarsNames,
+  );
+  const parentAvatar = useAppSelector(
+    state => state.user.parent?.avatar as AvatarsNames,
```
Comment (emberglaz) [inline app/screens/Menu/ManageProfiles/ManageProfiles.tsx]: here it should be immediately typed strictly
if it talks about the possibility of the null type, use the operator "?"

File: app/screens/Menu/ManageProfiles/components/UserImageTitle/UserImageTitle.types.ts [hunk 5]
```diff
@@ -0,0 +1,7 @@
+import { IMAGES } from '../../../../../assets';
+
+export interface IUserImageTitleProps {
+  image: keyof typeof IMAGES;
```
Comment (emberglaz) [inline app/screens/Menu/ManageProfiles/components/UserImageTitle/UserImageTitle.types.ts]: `AvatarsNameType`

File: app/assets/images/index.ts (hunk 3 above)
Comment (emberglaz) [reply app/assets/images/index.ts]: and modify IUser models

**Review by emberglaz — CHANGES_REQUESTED** [review]

**Review by Reykjavik151 — APPROVED** [review]

**Pull request merged**